src/groupeddataframe/splitapplycombine.jl

```diff
@@ -372,7 +389,7 @@ const KWARG_PROCESSING_RULES =
     """
 
 """
-    combine(gd::GroupedDataFrame, pairs...; keepkeys::Bool=true)
+    combine(gd::GroupedDataFrame, args...; keepkeys::Bool=true)
     combine(fun::Union{Function, Type}, gd::GroupedDataFrame; keepkeys::Bool=true)
     combine(pair::Pair, gd::GroupedDataFrame; keepkeys::Bool=true)
```

Comment:
we allow this:
1. to be less breaking (we allowed this in the past)
2. to be consistent with `map` (where we allow it as otherwise there would be no way to do "fast path" in `map` as it requires a `Pair` argument)

Reply:
OK, consistency with `map` is convincing. We could allow passing multiple pairs too, but that can be done later if needed.

Reply:
I disallowed this as conceptually `map` maps a single function.